en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,90 @@
+Customize Content Attribute Display
+===================================
+
+Content attributes type are mixed, it's difficult to render them to user. When we
```

Comment:
I agree with the need of an introduction but don't use 'we have added something'. Use instead 'something has been added'